sql/migrate/migrate.go

```diff
@@ -5,8 +5,11 @@
 package migrate
 
 import (
+	"bufio"
 	"bytes"
 	"context"
+	"crypto/sha256"
```

Comment:
wonder why `hash/fnv` is not enough.

Reply:
I chose SHA-256 because the `go.sum` does as well. https://go.dev/ref/mod#go-sum-files

Reply:
Good to know.